docs(tutorial/django): add note about unapplied migrations warning (#2346)
canonical/charmcraft: docs/tutorial/kubernetes-charm-django.rst

    Specifying ``0.0.0.0:8000`` allows for traffic outside of the Multipass VM.

>     When you run the command for the first time, you will get a warning about
>     unapplied migrations.
>     You can ignore this warning for now, as we are currently not performing any
>     database operations. We will set up a database in a later section.

Reviewer: ```suggestion
    When you run the command for the first time, you'll see a warning about
    unapplied migrations. We can ignore this warning for now, as we aren't
    currently performing any database operations. We'll set up the database later.
```

Making some minor adjustments, one code-related, the rest style-related.

The important bit here is contractions. We use contractions to make our tutorials more conversational, so they can flow more easily, like learning from a spoken interaction with an expert.

You can find this rule and others in the style guide. https://docs.ubuntu.com/styleguide/en/#contractions

Author: This is interesting. I didn't use contractions to avoid confusing non-native speakers or readers. Reading that section of the guide clarified it for me quite well. Thank you very much!

    Specifying ``0.0.0.0:8000`` allows for traffic outside of the Multipass VM.

>     When you run the command for the first time, you will get a warning about unapplied migrations.
>     You can ignore this warning for now, as we are currently not performing any database operations. We will set up a database in a later section.

Reviewer 2: ```suggestion
    When you run the command for the first time, you will get a warning about
    unapplied migrations.
    You can ignore this warning for now, as we are currently not performing any
    database operations. We will set up a database in a later section.
```
Breaking up your lines to avoid linting errors!

Author: Thank you for pointing this out!
